### PR TITLE
Add Sustainability Donation to Organisers guide

### PR DESCRIPTION
**Pull request by glasnt**
Majority approved by core organisers. 

**Executive summary**

Starting 1 January 2025, a modest, localised event fee will be introduced. This will be used solely to ensure a sustainable model to pay for the underlying infrastructure that powers the DevOpsDays global community. Both the global organisation and local events themselves will continue to be volunteer-run, and none of these funds will be used to remunerate volunteers at any level.

---

This PR proposes adding this to the organiser guide. After merging, blog post to come.

### CODE REVIEW

File: content/page/organizing.md [hunk 1]
```diff
@@ -207,6 +211,60 @@ First, a recommendation: Have a tiered budget for a Good, Better, Best. A Good v
...
+
+Today, we have the means and logistics in place to handle financial transactions in USD in the United States of America via ConferenceOps, and in official European currencies within the European Union via the Stichting DevOps Foundation. For this reason, we will be asking DevOpsDays events in those regions to participate in this sustainability programme from 1 January 2025. Events in other regions will be onboarded as we build out our capacity to manage them. 
+
+ * ConferenceOps: email Laura Wickett ([laura@conferenceops.com](laura@conferenceops.com)) for banking details
+ * Stichting DevOps Foundation [banking details](https://devops.foundation/)
+
+We actively encourage feedback on this from the local event organisers—especially from locations outside of our initial two regions. We further understand that local events may have no familiarity with such processes, and we encourage discussion on how we can make sure this is not an undue burden to your organising tasks.
+
+#### Sustainability Donation FAQs
```
Comment (glasnt) [inline content/page/organizing.md:237]: Having H3 headers makes the TOC look strange, reduce that. 

```suggestion
### Sustainability Donation FAQs
```

File: content/page/organizing.md [hunk 2]
```diff
@@ -207,6 +211,60 @@ First, a recommendation: Have a tiered budget for a Good, Better, Best. A Good v
...
 
+### Sustainability Donation
+
+🆕 New in 2025
+
+The management and operations of the DevOpsDays infrastructure has always been, and will continue to be, run on a volunteer basis. The services that provide this infrastructure, however, are not free. While we have benefitted from punctual donations and free tiers of services over the years, the reality is that this model is not sustainable. To be blunt: we’re running out of runway. In order to promote a sustainable funding model, and to ensure the continuity of the DevOpsDays global community, as of 1 January 2025, we are introducing a modest event fee: each time you run an event (generally once per year), you donate the equivalent of at least **one full price attendee ticket to your event** to the global DevOpsDays organisation (though you may choose to donate more, if you want).
```
Comment (phrawzty) [inline content/page/organizing.md:218]:
```suggestion
The management and operations of the DevOpsDays infrastructure has always been, and will continue to be, run on a volunteer basis. The services that provide this infrastructure, however, are not free. While we have benefitted from punctual donations and free tiers of services over the years, the reality is that this model is not sustainable. In order to promote a sustainable funding model, and to ensure the continuity of the DevOpsDays global community, as of 1 January 2025, we are introducing a modest event fee: each time you run an event (generally once per year), you donate the equivalent of at least **one full price attendee ticket to your event** to the global DevOpsDays organisation (though you may choose to donate more, if you want). 
```

File: content/page/organizing.md [hunk 3]
```diff
@@ -207,6 +211,60 @@ First, a recommendation: Have a tiered budget for a Good, Better, Best. A Good v
...
+
+Today, we have the means and logistics in place to handle financial transactions in USD in the United States of America via ConferenceOps, and in official European currencies within the European Union via the Stichting DevOps Foundation. For this reason, we will be asking DevOpsDays events in those regions to participate in this sustainability programme from 1 January 2025. Events in other regions will be onboarded as we build out our capacity to manage them. 
+
+ * ConferenceOps: email Laura Wickett ([laura@conferenceops.com](laura@conferenceops.com)) for banking details
+ * Stichting DevOps Foundation [banking details](https://devops.foundation/)
+
+We actively encourage feedback on this from the local event organisers—especially from locations outside of our initial two regions. We further understand that local events may have no familiarity with such processes, and we encourage discussion on how we can make sure this is not an undue burden to your organising tasks.
```
Comment (phrawzty) [inline content/page/organizing.md:235]:
```suggestion
```

File: content/page/organizing.md [hunk 4]
```diff
@@ -207,6 +211,60 @@ First, a recommendation: Have a tiered budget for a Good, Better, Best. A Good v
...
+
+This will help us pay for the following: 
+ * Cloud hosting for our conference services, including the content and ticketing management platforms, as well as the password manager
+ * Cloud hosting for our Mastodon instance, social.devopsdays.org
+ * Domain name registration for devopsdays.org
+ * Google Workspace for devopsdays.org for the event mailing lists and backend management
+
+We will continue to provide all of these tools and services to the entire DevOpsDays community. By accepting donations in this fashion, we can ensure a healthy future for events around the world, regardless of their location or situation.
+
+Today, we have the means and logistics in place to handle financial transactions in USD in the United States of America via ConferenceOps, and in official European currencies within the European Union via the Stichting DevOps Foundation. For this reason, we will be asking DevOpsDays events in those regions to participate in this sustainability programme from 1 January 2025. Events in other regions will be onboarded as we build out our capacity to manage them.
```
Comment (phrawzty) [inline content/page/organizing.md:230]:
```suggestion
Currently, we have the means and logistics in place to handle financial transactions in USD in the United States of America via ConferenceOps, and in official European currencies within the European Union via the Stichting DevOps Foundation. For this reason, we will be asking DevOpsDays events in those regions to participate in this sustainability programme from 1 January 2025. Events in other regions will be onboarded as we build out our capacity to manage them. 
```

**Review by phrawzty — CHANGES_REQUESTED** [review]
Minor changes since this is long term doc and not a punctual email.

File: content/page/organizing.md [hunk 5]
```diff
@@ -98,6 +98,8 @@ The code of conduct must include the following:
   - Why: Marketing is tricky, especially as a new event in a world where social media is quickly changing. Be sure to check out the [Visibility](#visibility) section for some tips and tricks.
 * **T-6.5 Months to Event:** Ensure registration opens
   - Why: The more time the registration is open, the more potential visibility it gets. Be sure to checkout the [Setting Up Registration](#setting-up-registration) section for more information.
+* **T-5 months before Event:** Pay sustainability donation
```
Comment (phrawzty) [inline content/page/organizing.md:101]:
```suggestion
* **T-5 months before Event:** Contribute "sustainability donation"
```

**Review by phrawzty — CHANGES_REQUESTED** [review]
Oops, missed this one in the first pass.

**Review by phrawzty — APPROVED** [review]

**Pull request merged**